framework/Models/PostProcessor.py

```diff
@@ -188,10 +191,10 @@ def evaluateSample(self, myInput, samplerType, kwargs):
           the second item will be the output of this model given the specified
           inputs
     """
-    Input = self.createNewInput(myInput,samplerType, **kwargs)
-    if Input is not None and len(Input) == 0:
-      Input = None
-    returnValue = (Input, self.run(Input))
+    ppInput = self.createNewInput(myInput,samplerType, **kwargs)
```

Comment:
I am conflicted here again. the samplerType is set at the Step stage...so for PostProcessor steps it will always be None. I dnk but I think it is not needed and it can cause confusion for the PostProcessor developer

Reply:
I know it was there arleady, but probably it is a good time to remove it

Reply:
I agree to remove it.

Reply:
I have updated the docstring for better clarification. I think we can keep "samplerType" which will consistent with other Models at entity level. In addition, these methods will only at entity level, and we will not expose this method to post-processor developer. In other words, the pp developers will not see these methods, so there is no confusion for them.

Reply:
Please open under discussion issue.